src/server/main_service.cc

```diff
@@ -1367,6 +1381,11 @@ bool Service::InvokeCmd(const CommandId* cid, CmdArgList tail_args, ConnectionCo
     return false;
   }
 
+  auto reason = cntx->reply_builder()->ConsumeLastError();
+  if (!reason.empty()) {
+    LOG_EVERY_T(WARNING, 1) << FailedCommandToString(cid->name(), tail_args, reason);
```

Comment:
The only downside is that this is not properly logging `multi/exec` commands because of the `1second` delay. Any preferences @romange ?

Reply:
can you give an example?

Reply:
`
multi
set foo bar lalala
set bar foo lalala
exec
```

Only logs `set foo bar lalala` but omits `set bar foo lalala`